gulp/tasks/e2eTests.js

```diff
@@ -0,0 +1,28 @@
+module.exports = function (gulp, config, plugins) {
+
+    gulp.task('connectDist', function (done) {
+        plugins.connect.server({
+            root: 'dist/',
```

Comment:
Do we run tests on 'dist' or 'build'? Also use config for paths. (example config.distPath)